### PR TITLE
added detailed instructions on upgrading control plane and edge gatew…

### PR DESCRIPTION
**Pull request by LLe27**
…ays on tyk cloud

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

**Comment by letzya** [comment]
@LLe27 FYI this needs peer review

**Comment by LLe27** [comment]
Thanks @letzya , submitted to slack for approval.

**Comment by letzya** [comment]
@LLe27 can we please get this reviewed and approved?

**Comment by LLe27** [comment]
@letzya i'll reach out to some people and see if anyone can review

File: tyk-docs/content/tyk-cloud/environments-&-deployments/managing-gateways.md [hunk 1]
```diff
@@ -60,3 +60,25 @@ To edit an existing Edge Gateway:
...
+
+{{< img src="/img/admin/cp-edge-upgrade-version.png" alt="Bundle version drop-down" >}}
+
+4. To apply your changes, click the **"Save and Re-Deploy"** button located at the top right. After a few seconds, you will be redirected to the overview page of the Control Plane and a **"Deploying"** indicator button will appear. 
+
+{{< img src="/img/admin/cp-edge-upgrade-deploying.png" alt="Deploying notification" >}}
+
+5. A **"Deployed"** button indicates a successful upgrade.
+
+{{< img src="/img/admin/cp-edge-upgrade-deployed.png" alt="Deploying notification" >}}
```
Comment (vverbani) [inline tyk-docs/content/tyk-cloud/environments-&-deployments/managing-gateways.md]:
```suggestion
{{< img src="/img/admin/cp-edge-upgrade-deployed.png" alt="Deployed notification" >}}
```

File: tyk-docs/content/tyk-cloud/environments-&-deployments/managing-control-planes.md [hunk 2]
```diff
@@ -59,4 +59,25 @@ To edit an existing Control Plane:
...
+
+{{< img src="/img/admin/cp-edge-upgrade-version.png" alt="Bundle version drop-down" >}}
+
+4. To apply your changes, click the **"Save and Re-Deploy"** button located at the top right. After a few seconds, you will be redirected to the overview page of the Control Plane and a **"Deploying"** indicator button will appear. 
+
+{{< img src="/img/admin/cp-edge-upgrade-deploying.png" alt="Deploying notification" >}}
+
+5. A **"Deployed"** button indicates a successful upgrade.
+
+{{< img src="/img/admin/cp-edge-upgrade-deployed.png" alt="Deploying notification" >}}
```
Comment (vverbani) [inline tyk-docs/content/tyk-cloud/environments-&-deployments/managing-control-planes.md]:
```suggestion
{{< img src="/img/admin/cp-edge-upgrade-deployed.png" alt="Deployed notification" >}}
```

**Review by vverbani — CHANGES_REQUESTED** [review]

**Review by vverbani — APPROVED** [review]

**Pull request merged**